tests/blackbox_test.go

```diff
@@ -230,14 +230,16 @@ func outer(t *testing.T, test types.Test, negativeTests bool) {
 		}
 	}
 
-	// Ignition config
+	// Ignition configs
 	if err := ioutil.WriteFile(filepath.Join(tmpDirectory, "config.ign"), []byte(test.Config), 0666); err != nil {
 		t.Fatal(err)
 	}
+	profilePath := filepath.Join(tmpDirectory, "profile")
```

Comment:
I'd prefer for this to default to `""` if no profile specific information is present.

Reply:
The profile is also needed for any test that uses `oem:///` URLs, and possibly for other things in the future, so autodetecting this case seems a bit error-prone.  Is it worthwhile to add a `Test` flag that enables use of the profile?

Reply:
A flag on the `Test` object would be fine. I'd just prefer that any tests which are not specifying the profile hit the default case where the Profile is built by ignition.